intuitlib/utils.py

```diff
@@ -165,11 +164,12 @@ def validate_id_token(id_token, client_id, intuit_issuer, jwk_uri):
         return False
 
     message = id_token_parts[0] + '.' + id_token_parts[1]
```

Comment:
Can remove the following as they are no longer used:

- `id_token_signature = urlsafe_b64decode(((_correct_padding(id_token_parts[2])).encode('ascii')))`
- `message = id_token_parts[0] + '.' + id_token_parts[1]`